skabase/SKALogger/setup.py

```diff
@@ -20,8 +20,7 @@
 with open(readme_filename) as file:
     long_description = file.read()
 
-# release_filename = os.path.join(setup_dir, 'SKALogger', 'release.py')
-release_filename = os.path.join(setup_dir,  'release.py')
```

Comment:
Still need to delete the unused file:  `skabase/SKALogger/release.py`

Reply:
This is outdated. I deleted it, it was giving error. So I kept the file and kept the previous line of code..
`release_filename = os.path.join(setup_dir, 'SKALogger', 'release.py')`

Reply:
skabase/SKAxyz/release.py exist in all other modules too, with
`release_filename = os.path.join(setup_dir, 'SKAxyz', 'release.py')`

Are they all unused?

Reply:
I think so.  That files that will be used are `skabase/SKAxyz/SKAxyz/release.py`
The `setup_dir` will be `skabase/SKAxyz`

Reply:
Will have to look at it.